cms/templates/group_configurations.html

```diff
@@ -47,32 +49,45 @@ <h1 class="page-header">
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-        </div>
 
-      % if should_show_experiment_groups:
-        <div class="wrapper-groups experiment-groups">
-          <h3 class="title">${_("Experiment Group Configurations")}</h3>
-          % if experiment_group_configurations is None:
-            <div class="notice notice-incontext notice-moduledisabled">
-                <p class="copy">
-                    ${_("This module is disabled at the moment.")}
-                </p>
-            </div>
-          % else:
-            <div class="ui-loading">
-                <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
-            </div>
-          % endif
-        </div>
-      % endif
+        % for config in all_group_configurations:
+          <div class="wrapper-groups content-groups ${config['scheme']}">
+            <h3 class="title">${config['name']}</h3>
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+          </div>
+        % endfor
+
+        % if should_show_experiment_groups:
+          <div class="wrapper-groups experiment-groups">
+            <h3 class="title">${_("Experiment Group Configurations")}</h3>
+            % if experiment_group_configurations is None:
+              <div class="notice notice-incontext notice-moduledisabled">
+                  <p class="copy">
+                      ${_("This module is disabled at the moment.")}
+                  </p>
+              </div>
+            % else:
+              <div class="ui-loading">
+                  <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+              </div>
+            % endif
+          </div>
+        % endif
       </article>
       <aside class="content-supplementary" role="complementary">
+      % if should_show_enrollment_track:
+        <div class="bit">
+          <div class="enrollment-track-doc">
+            <h3 class="title-3">${_("Enrollment Track Content Groups")}</h3>
+            <p>${_("Enrollment track configurations are added based on the Course Modes configured for your course. Learners within a given Course Mode will be added to the corresponding enrollment track group automatically. The enrollment track groups can be used like the manually created content groups to set content visibility.")}</p>
+            <p><a href="${get_online_help_info(enrollment_track_help_token())['doc_url']}" target="_blank" class="button external-help-button">${_("Learn More")}</a></p>
+          </div>
+        </div>
+      % endif
```

Comment:
@staubina I'm not able to add comments at the exact lines, but in the sidebar text for Content Groups and Experiment Groups, can we change "students" to "learners"? I think there are just 2 instances: line 95 and line 104. Thanks!